src/js/msp/MSPHelper.js

```diff
@@ -1278,9 +1278,77 @@ MspHelper.prototype.process_data = function(dataHandler) {
                     TRANSPONDER.data.push(data.readU8());
                 }
                 break;
+
             case MSPCodes.MSP_SET_TRANSPONDER_CONFIG:
                 console.log("Transponder config saved");
                 break;
+
+            case MSPCodes.MSP_VTX_CONFIG:
+
+                VTX_CONFIG.vtx_type = data.readU8();
+                VTX_CONFIG.vtx_band = data.readU8();
+                VTX_CONFIG.vtx_channel = data.readU8();
+                VTX_CONFIG.vtx_power = data.readU8();
+                VTX_CONFIG.vtx_pit_mode = data.readU8() != 0;
+                VTX_CONFIG.vtx_frequency = data.readU16();
+                VTX_CONFIG.vtx_device_ready = data.readU8() != 0;
+                VTX_CONFIG.vtx_low_power_disarm = data.readU8();
+
+                if (semver.gte(CONFIG.apiVersion, "1.42.0")) {
+                    VTX_CONFIG.vtx_pit_mode_frequency = data.readU16();
+                    VTX_CONFIG.vtx_table_available = data.readU8() != 0;
+                    VTX_CONFIG.vtx_table_bands = data.readU8();
+                    VTX_CONFIG.vtx_table_channels = data.readU8();
+                    VTX_CONFIG.vtx_table_powerlevels = data.readU8();
+                }
+                break;
+
+            case MSPCodes.MSP_SET_VTX_CONFIG:
+                console.log("VTX config saved");
```

Comment:
I wouldn't use 'saved' here - the command only sets the config, EEPROM still needs to be saved.